nf1_cellprofiler_classifier_fs_all_cellprofiler_all_feat_perfect.ipynb

```diff
@@ -0,0 +1,499 @@
+{
```

Comment:
<p>What do the values in the boxes mean?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/NF1_SchwannCell_data_analysis/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Those give you an understanding of how the model performs on n classes. The grid will be n x n.

Here is a good article on it: 
https://www.sciencedirect.com/topics/engineering/confusion-matrix

If you want me to explain it though, just let me know.